src/components/util/modifications-type.js

```diff
@@ -0,0 +1,59 @@
+export const MODIFICATION_TYPES = {
+    GROOVY_SCRIPT: {
+        type: 'GROOVY_SCRIPT',
```

Comment:
Why using a new nested field "type", because in the future more fields will be added ?
If so, we should rename "MODIFICATION_TYPES" into "MODIFICATIONS" or "NETWORK_MODIFICATIONS", otherwise I don't see the point of adding a field "type" if it only represent the type.

Reply:
I think it's better to do it in this PR, @SlimaneAmar what do you think ?

Reply:
I'm agree with you , it will just require the validation of Slimane